sources/packages/backend/apps/db-migrations/src/sql/Institution/Add-col-other-regulating-body.sql

```diff
@@ -0,0 +1,4 @@
+-- Add other_regulating_body column to institutions table
+ALTER TABLE sims.institutions ADD COLUMN IF NOT EXISTS other_regulating_body VARCHAR(100);
```

Comment:
looks like you don't have the SQL formater

Reply:
same for all .sql files

Reply:
Yes @ann-aot is right. It looks like this if formatted with the sql formatter. 
![image](https://user-images.githubusercontent.com/54600590/221716216-a3c30a8a-d372-440e-98e5-53e5ffe7c302.png)

Reply:
Thank You Ann and Deepak. Fixed it.